packages/plugin-dev/src/browser/hosted-plugin-manager-client.ts

```diff
@@ -68,7 +68,7 @@ export namespace HostedPluginCommands {
 export enum HostedInstanceState {
     STOPPED = 'stopped',
     STARTING = 'starting',
-    RUNNNING = 'running',
```

Comment:
This is a breaking change, can you reflect so in the `changelog`?

Reply:
done